features/check-file-contents.feature

```diff
@@ -107,3 +107,33 @@ Feature: Check files in a WordPress install
       """
       [{"name":"file-server-name-wp-config","status":"error","message":"1 'php' file failed check for 'define\\(.+WP_(HOME|SITEURL).+\\$_SERVER.+SERVER_NAME'."}]
       """
+
+  Scenario: Check for the successful use of the exist flag
+    Given a WP install
+    And a config.yml file:
+      """
+      file-content-exist:
+        check: File_Contents
+        options:
+          regex: .*wp-doctor-exists-test.*
+          only_wp_content: true
+          exists: true
+      """
+
+    When I run `wp doctor check file-content-exist --config=config.yml --format=json`
+    Then STDOUT should be JSON containing:
+      """
+      [{"name":"file-content-exist","status":"error","message":"0 'php' files failed check for '.*wp-doctor-exists-test.*'."}]
```

Comment:
The count appears to be correct but the messaging is not, which I believe is causing the error. Looking into it now